apps/Conduit-UI/src/features/storage/components/StorageConfig.tsx

```diff
@@ -186,6 +186,12 @@ const StorageConfig: React.FC = () => {
                         label="Account Id"
                       />
                     </Grid>
+                    <Grid item md={6} xs={12}>
```

Comment:
Could you add a text before this input that explains that the endpoint option should be left empty if using AWS S3 or filled with an endpoint if using an S3-compatible provider?

Reply:
Definitely can. I wondered about that since it will still work if an AWS S3 user adds the S3 endpoint in there?  I guess by stating that, it simplifies the process for AWS S3 users so that they don't have to hunt down the AWS S3 endpoint